### ISSUE TITLE
Human Interface Guidelines

### ISSUE THREAD

**Issue by Secret-chest** [linuxmint/linuxmint#444]
We need HIGs to be a platform, and in today's world, this is very important.
I have started writting some, can you consider them? (I will work on them more)

**Comment by tripleo1** [comment]
This work is interesting.

**Comment by BeatLink** [comment]
Is there any further work on developing this?

**Comment by Secret-chest** [comment]
No, because I think what I did first was silly, but we need guidelines, and possibly our own fork of GTK, so more apps will be made.
